ReactNativeClient/lib/BaseApplication.js

```diff
@@ -170,6 +170,14 @@ class BaseApplication {
 				continue;
 			}
 
+			// Electron-specific flag used for debugging - ignore it
+			if (arg.includes('--remote-debugging-port')) {
```

Comment:
Why "includes"? Can't it just be equality like the other flags?

Reply:
Currently Electron only accepts this flag in the '--x=y' form so the arg string will contain the additional parameter.

Reply:
Ok I see. In this case, and since the port is required, I'd just check that `arg.indexOf('--remote-debugging-port=') === 0'` and always skip one argument. I prefer all these checks to be strict so that in case of an issue we get a better error message.